ext/opentelemetry-ext-flask/tests/test_flask_integration.py

```diff
@@ -93,14 +106,7 @@ def test_404(self):
 
         self.assertEqual(
             self.span_attrs,
-            {
-                "component": "http",
-                "http.method": "POST",
-                "http.host": "localhost",
-                "http.url": "http://localhost/bye",
-                "http.status_code": 404,
-                "http.status_text": "NOT FOUND",
-            },
+            {"http.status_code": 404, "http.status_text": "NOT FOUND"},
```

Comment:
random and more general question: is it needed to have "status_text"? feels to me like it adds a lot to the payload and is often identical.

Reply:
No, the status text is optional. But note that the integration does not insert static text here but actual "output" from the WSGI application.